examples/diagnostics_and_criticism/model_averaging.ipynb

```diff
@@ -2,14 +2,8 @@
  "cells": [
```

Comment:
<p>Not sure why there is a mean after the call to hdi, I would suggest doing:</p><p> </p><pre class="ql-syntax" spellcheck="false">hpd_w = az.hdi(ppc_w["kcal"].flatten())
</pre><p> </p><p>instead. If I'm not wrong, between this and the changes to <code>sample_...</code>  there won't be a need to ignore warnings anymore everything will have been solved and will follow best practices</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>